server/graphql/v2/input/UpdateCreateInput.js

```diff
@@ -10,6 +10,7 @@ export const UpdateCreateInput = new GraphQLInputObjectType({
   fields: () => ({
     title: { type: new GraphQLNonNull(GraphQLString) },
     isPrivate: { type: GraphQLBoolean },
+    isChangelog: { type: new GraphQLNonNull(GraphQLBoolean) },
```

Comment:
I think it's fine to allow people to create updates without providing a value for `isChangelog` here since it only affects the team and defaults to `false`. That will also make sure that the form does not crash if someone tries to submit an update with an older frontend.

```suggestion
    isChangelog: { type: GraphQLBoolean },
```